Revise documentation for npm run commands (#501)
EthanThatOneKid/acmcsuf.com: CONTRIBUTING.md

## Submitting your own Contributions 🎉

Before publishing a PR, it is required that you run `npm run all` to make sure that you are complying with our style guidelines and passing all checks.

```bash

> # Runs the autoformatter, WARNING: this might mutate your files.
> npm run format
> # Runs the linter, WARNING: this might mutate your files.
> npm run lint
> # Runs the checks for type errors, unused css, and more (See: https://github.com/sveltejs/language-tools/tree/master/packages/svelte-check)
> npm run check
> # Run all our unit tests (or with `npm t`)
> npm run test
> # Builds the website
> npm run build

Reviewer: Nit: Please separate these lines to make it clear which comment is intended for which command.
```suggestion
# Runs the autoformatter, WARNING: this might mutate your files.
npm run format

# Runs the linter, WARNING: this might mutate your files.
npm run lint

# Runs the checks for type errors, unused css, and more (See: https://github.com/sveltejs/language-tools/tree/master/packages/svelte-check)
npm run check

# Run all our unit tests (or with `npm t`)
npm run test

# Builds the website
npm run build
```

Author: addressed

> 

Reviewer: Nit: Extra line break.

Author: addressed

# Runs all of the above checks necessary for a release deployment

> npm run release

Reviewer: Script `release` is not in the final `package.json` file.

Author: addressed

> npm run release

Reviewer: ```suggestion
npm run all
```

> # Runs the above and hosts the production version locally

Reviewer: ```suggestion
# Runs `npm run all` and hosts the production version locally
```

Author: addressed
